release-notes-3.1.1.md

```diff
@@ -0,0 +1,53 @@
+# Release Notes
+
+This release makes no changes to the OpenAPI standard, however the formal specification document had multiple improvements, including:
+- clarity and expanded explanations including several new appendices with supplementary details
+- examples and additional documentation moved to https://learn.openapis.org
+- the HTML rendering of the specification at https://spec.openapis.org is now considered the authoritative version (formerly it was the Markdown source on GitHub)
+
+OpenAPI description writers should mark their OpenAPI descriptions with the version of OpenAPI specification they used to write their specification, updating where possible.
+
+Tool creators should also not expect changes, but it is recommended to check the list of changes below.
+
+## Clearer Definitions
+
+Introduce consistent language around OpenAPI Document/Description/Definition:
+- OpenAPI Description means the OpenAPI description of an API, whether it is in one or many files.
+- a document means a single file.
+- an "entry document" is where the OpenAPI description for an API starts; it may reference other documents.
```

Comment:
```suggestion
- A document means a single file.
- An "entry document" is where the OpenAPI description for an API starts; it may reference other documents.
```
All list items end with a period